cypress/integration/project/source_browsing.spec.ts

```diff
@@ -191,33 +191,6 @@ context("project source browsing", () => {
               )
               .should("exist");
 
-            // Going to a different path and then switching back to the root path
-            // shows the README again.
-            commands.pick("source-tree").within(() => {
-              cy.contains(".i-am-well-hidden").click();
-            });
-            commands
-              .pick("project-screen", "file-view")
-              .contains("platinum")
-              .click();
-            commands
-              .pickWithContent(["project-screen", "file-view"], "README.md")
-              .should("exist");
-
-            commands.pick("source-tree").within(() => {
-              cy.contains(".i-too-am-hidden").click();
-            });
-            commands
-              .pickWithContent(
-                ["project-screen", "file-view"],
-                "platinum / .i-too-am-hidden"
-              )
-              .should("exist");
-            commands.pick("project-screen", "file-view", "root-link").click();
-            commands
-              .pickWithContent(["project-screen", "file-view"], "README.md")
-              .should("exist");
-
```

Comment:
I'd rather we keep the test, but just click on the "Files" tab to get back to root:

```suggestion
// Going to a different path and then switching back to the root path
// shows the README again.
commands.pick("source-tree").within(() => {
  cy.contains(".i-am-well-hidden").click();
});
commands.pick("project-screen", "files-tab").click();
commands
  .pickWithContent(["project-screen", "file-view"], "README.md")
  .should("exist");

commands.pick("source-tree").within(() => {
  cy.contains(".i-too-am-hidden").click();
});
commands
  .pickWithContent(
    ["project-screen", "file-view"],
    ".i-too-am-hidden"
  )
  .should("exist");
commands.pick("project-screen", "files-tab").click();
commands
  .pickWithContent(["project-screen", "file-view"], "README.md")
  .should("exist");
```

Reply:
:+1:

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1609/commits/dbcfebd6c09972c1aceb4a5f4a8e62ea674120ac